bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/ReadRowsRequestManager.java

```diff
@@ -67,77 +47,18 @@
    * <p>
    * Constructor for ResumingStreamingResultScanner.
    * </p>
-   * @param retryOptions a {@link RetryOptions} object.
    * @param originalRequest a {@link ReadRowsRequest} object.
-   * @param rpcMetrics a {@link com.google.cloud.bigtable.grpc.async.BigtableAsyncRpc.RpcMetrics} object to keep track of retries and
-   *          failures.
-   * @param logger a {@link Logger} to log info messages about the state of retries.
    */
-  public ReadRowsRequestRetryHandler(RetryOptions retryOptions, ReadRowsRequest originalRequest,
-      RpcMetrics rpcMetrics, Logger logger) {
+  public ReadRowsRequestManager(ReadRowsRequest originalRequest) {
```

Comment:
Might be a good idea to make this class package protected to signify that its not part of the public api